### DISCUSSION TITLE
What is the "fault" in a pod failure experiment?

### DISCUSSION THREAD

**Discussion by bruteMax** [chaos-mesh/chaos-mesh#2982] [Q&A] [answered]
The docs say that a pod failure injects "fault" to make the targeted pod unavailable, but what exactly is that fault?  

Thanks,

Bill.

**Comment by STRRL** [comment, marked as answer]
The implementation of pod-failure chaos experiment would "patch" the `image` field, replace it with `gcr.io/google-containers/pause:latest` (you could change it in helm value `controllerManager.podChaos.podFailure.pauseImage`).

So target container would still keep running, but it would provide service anymore.

**Reply by bruteMax** [reply]
Thank you!
